README.md

```diff
@@ -37,8 +34,8 @@
 `@storyblok/svelte` helps you connect your Svelte project to Storyblok by:
 
 - Providing the `getStoryblokApi` function to interact with the Storyblok APIs, using the [storyblok-js-client](https://github.com/storyblok/storyblok-js-client)
-- Enabling real time editing through the [Storyblok Bridge](https://www.storyblok.com/docs/Guides/storyblok-latest-js?utm_source=github.com&utm_medium=readme&utm_campaign=storyblok-svelte)
-- Providing the `StoryblokComponent` which allows you to connect your components to the Storyblok Visual Editor
+- Enabling real-time editing through the [Storyblok Bridge](https://www.storyblok.com/docs/Guides/storyblok-latest-js?utm_source=github.com&utm_medium=readme&utm_campaign=storyblok-svelte)
+- Providing the `StoryblokComponent`, which allows you to connect your components to the Storyblok Visual Editor
```

Comment:
We could link the Visual Editor Docs here: https://www.storyblok.com/docs/editor-guides/visual-editor